ios/RNModules/RNLogging/RNLogging/RNLoggingModule.m

```diff
@@ -1,22 +1,8 @@
 #import "RNLoggingModule.h"
+#import "RNLogger.h"
 
 #import <React/RCTBridgeModule.h>
 
-@interface RNDummyLogger: NSObject <RNLogger>
```

Comment:
This was still useful for us to see whether the right logging was called at the right time when developing.

Reply:
Sorry, I did not want to remove something you are using... I implemented it again though a bit differently.